lib/connect.js

```diff
@@ -354,6 +354,11 @@ function connect(token, options) {
         options.clientTrackSwitchOffControl = 'auto';
       }
 
+      if ('maxTracks' in options.bandwidthProfile.video) {
+        // move maxTracks over to new property that will be used to create RSP payload
+        options.bandwidthProfile.video.maxSwitchedOnTracks = options.bandwidthProfile.video.maxTracks;
```

Comment:
Aren't we removing maxTracks anyway so we don't need this check?

Reply:
`maxTracks` is marked as deprecated for now. But yes we do need to stop supporting this and  few other properties and remove related code. but that can come later.